src/libs/ajax/GoogleStorage.ts

```diff
@@ -76,7 +76,10 @@ const checkRequesterPaysError = async (response): Promise<RequesterPaysErrorInfo
 };
 
 export const responseContainsRequesterPaysError = (responseText) => {
-  return _.some((msg) => _.includes(msg, responseText), ['access to user project', 'user project to bill']);
+  return _.some(
+    (msg) => _.includes(msg, responseText),
+    ['access to user project', 'user project to bill', 'requester pays bucket']
```

Comment:
TODO: The `checkBucketReadAccess` requester pays error also happens on workspaces where the user is a writer/owner. Find a workaround that does not involve skipping `checkBucketReadAccess`.